packages/hash/api/src/model/entity.model.ts

```diff
@@ -535,11 +536,23 @@ class __Entity {
           const systemAccountId = await client.getSystemAccountId();
 
           const name = capitalizeComponentName(componentId);
+
+          // ensure a trailing a trailing slash on componentId
+          const componentIdUrl = new URL(
+            "./block-schema.json",
+            `${componentId.replace(/\/+$/, "")}/`,
+          );
+
+          const blockSchema = await (await fetch(componentIdUrl.href))
+            .json()
+            .catch((_) => ({}));
```

Comment:
Is this underscore needed? 

```suggestion
            .catch(() => ({}));
```